### DISCUSSION TITLE
OSM, Overture, Foursquare, POIs, ...

### DISCUSSION THREAD

**Discussion by vng** [organicmaps/organicmaps#9701] [Ideas]
Foursquare recently open-sourced its POIs. So I have a general question here.

Did anybody see some trustworthy articles or tools to check _the quality_ of Overture POIs data? Idealy some merged "planet" files with OSM and _good_ POIs from Overture (and now from Foursquare too).

Saying quality I mean good coordinates, not very outdated, not a spam ..

@Zverik @RedAuburn @pastk @map-per @patepelo @matheusgomesms 

Feel free to tag more relevant people here.

**Comment by matheusgomesms** [comment]
Overture has this "confidence" property, and some studies indicate that using a high threshold might remove lots of irrelevant POIs, making it usable (https://observablehq.com/d/9847c08c46f56ed6).

Also here: https://www.openstreetmap.org/user/mikelmaron/diary/402600

I haven't checked the Foursquare data yet, but @RedAuburn initiated a discussion: https://community.openstreetmap.org/t/foursquare-releases-100m-poi-dataset-under-apache-2-0/121883

I would be very cautions to use these datasets, but I think it's doable. Lots of pros and cons.

**Reply by patepelo** [reply]
Apple Maps uses Foursquare data for a reason, I suppose it's strongly dependant on the region

**Comment by vng** [comment]
Did anybody try to translate or classify Overture POIs (categories) scheme into the OSM scheme?

**Reply by matheusgomesms** [reply]
https://github.com/whubsch/overturetoosm

**Comment by pastk** [comment]
CC @matkoniecz @zyphlar

**Comment by hb0nd** [comment]
a small thread on the Foursquare data quality (not positive): https://en.osm.town/@foxy/113516697493947980

**Comment by map-per** [comment]
Westnordost once did an small analysis: https://github.com/streetcomplete/StreetComplete/issues/5199#issuecomment-1681958518 (_but that was before Foursquare data was released_)

**Comment by map-per** [comment]
I don't like using other datasets in addition to OSM, because it causes problems in the editor. Using OM for editing doesn't really work when just some of the data is form OSM and some not. So users might e.g end up reporting an OSM POI as "Place doesn't exist" because it is a duplicate of an Overture POI that wasn't filtered out in conflation.

Other problems could be similar to #9258. OM allows editing of Tiger addresses in the US but uploading doesn't work.

**Comment by zyphlar** [comment]
Yeah the purity of using a fully open source dataset is very valuable. The only reason I'm in favor of alternate datasets for addresses is because that's a fundamental global fallback necessary for navigation, but I'm not convinced that alternate sources of POIs are a great thing to just "import"/overlay inside OM. If anything, if these sources are truly open, maybe some projects could seek to import them into OSM directly? A truly open project like OSM should be as authoritative as possible for all these things in my opinion. Then business owners and users can get used to updating things there and everyone can benefit: Google, Apple, Facebook, and otherwise. Overture seems like a corporate band-aid to me.

**Comment by vng** [comment]
@RedAuburn From your link :)
```
- POIs themselves are in the best traditions MAPS.ME. Foursquare users also add personal notes to the map.

- By the way, you can view Foursquare data in... MAPS.ME. You can enable the Foursquare POI layer there
```

I've checked MM with Foursquare in my neighborhood - rubbish ..

**Comment by matkoniecz** [comment]
I heard from some that Overture POI data, after heavy filtering is usable in USA.

But in Poland it is an utter disaster, with hilariously bogus, misclassified, misplaced, wrong, outdated, spam, duplicated, tripclicates, fourplicates and so on. Note: I did a cursory check and closed tab, it was so hilariously bad. Filtering helped a bit, but heavy bogus data never went away.

I am heavy OSM user so I prefer less POIs and be able to add missing ones over having also bunch of incorrect, misplaced or wrong/outdated and not being able to fix this.

**Comment by zyphlar** [comment]
I would be okay with a sort of crowd sourced POI routine. For example let's say you're searching for McDonald's but there aren't many results nearby. Overture results could be shown as a backup. Then if the user selects that result (and maybe sees its location on the map, and/or navigates to it) OM could ask "is this McDonald's location accurate?" and if so queued for addition or a note added to OSM for "there's likely a McDonald's here."

Likewise Overture POIs could be shown on the map only if there are no other POIs in the area (as backups) but as others have said it would be very easy to "pollute" the map with low quality outdated information generating complaints. Ultimately simply making OM usable and popular will help POIs be added worldwide: Google already uses this approach, they're just much more popular.
